app/models/escort.rb

```diff
@@ -9,20 +9,18 @@ def self.find_detainee_by_prison_number(number)
     joins(:detainee).where(detainees: { prison_number: number }).first
   end
 
-  def move
-    super || build_move
-  end
-
-  def healthcare
-    super || build_healthcare
+  def self.create_with_children(prison_number:)
+    new.tap do |escort|
+      escort.build_detainee(prison_number: prison_number)
+      escort.build_healthcare
+      escort.build_offences
+      escort.build_risks
+      escort.save!
+    end
   end
 
-  def offences
-    super || build_offences
-  end
-
-  def risks
-    super || build_risks
+  def move
```

Comment:
It's got different semantics. Not quite sure what they are yet, but it doesn't fall into the exact same category as Risks or Healthcare.